test/tests/unit/wire_protocol_test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bson = require('bson');
+const wireProtocol2_6 = require('../../../lib/wireprotocol/2_6_support.js');
+const wireProtocol3_2 = require('../../../lib/wireprotocol/3_2_support.js');
+
+const MockPool = {};
+
+MockPool.write = function(cmd, options, callback) {
+  callback(null, cmd);
+};
+
+describe('WireProtocol', function() {
+  it('2.6 should only set bypassDocumentValidation to true if explicitly set by user to true', function(done) {
+    const options = { bypassDocumentValidation: true };
+    wireProtocol2_6.prototype.insert(MockPool, '', '', bson, 'ops', options, function(err, data) {
```

Comment:
finally, I think for readability-sake in the future it would be best to name the variables you are passing in fake data for, e.g. 
```
const ns = 'fake.namespace';
const ismaster = {};
...

wireProtocol.insert(pool, ns, ismaster, ...)
```